jms/src/main/scala/akka/stream/alpakka/jms/Jms.scala

```diff
@@ -15,6 +17,19 @@ sealed trait Destination
 final case class Topic(name: String) extends Destination
 final case class Queue(name: String) extends Destination
 
+final case class JmsTextMessage(body: String, properties: Map[String, Any] = Map.empty) {
```

Comment:
Could you quickly explain what's the reason to introduce this new type instead of using the JMS message type directly?